releasenotes/notes/0.14/0.14.0-release-19557dcd9d5af6e3.yaml

```diff
@@ -0,0 +1,71 @@
+  * boolean_logic: circuits that transform basis state according to simple
+    Boolean logic functions, such as :class:`~qiskit.circuit.library.ADD` or
+    :class:`~qiskit.circuit.library.XOR`.
+  * arithmetic: a set of circuits for doing classical arithmetics such as
+    :class:`~qiskit.circuit.library.WeightedAdder` and
+    :class:`~qiskit.circuit.library.IntegerComparator`.
+  * basis_changes: circuits such a the quantum Fourier transform,
+    :class:`~qiskit.circuit.library.QFT`, that mathematically apply basis
+    changes.
+  * n_local: patterns to easily create large circuits with rotation and
+    entanglement layers, such as  :class:`~qiskit.circuit.library.TwoLocal`
+    which uses single-qubit rotations and two-qubit entanglements.
+  * data_preparation: circuits that take classical input data and encode it
+    in a quantum state that is difficult to simulate, e.g.
+    :class:`~qiskit.circuit.library.PauliFeatureMap` or
+    :class:`~qiskit.circuit.library.ZZFeatureMap`.
+  * Other circuits that have proven interesting in the literature, such as
+    :class:`~qiskit.circuit.library.QuantumVolume`,
+    :class:`~qiskit.circuit.library.GraphState`, or
+    :class:`~qiskit.circuit.library.IQP`.
+  
+  To allow easier use of these circuits as building blocks, we have introduced
+  a :meth:`~qiskit.circuit.QuantumCircuit.compose` method for composition of
+  circuits either with other circuits (by welding them at the ends and
+  optionally permuting wires) or with other simpler gates::
+  
+    >>> lhs.compose(rhs, qubits=[3, 2], inplace=True)
+  
+  .. parsed-literal::
+                    ┌───┐                   ┌─────┐                ┌───┐
+        lqr_1_0: ───┤ H ├───    rqr_0: ──■──┤ Tdg ├    lqr_1_0: ───┤ H ├───────────────
+                    ├───┤              ┌─┴─┐└─────┘                ├───┤
+        lqr_1_1: ───┤ X ├───    rqr_1: ┤ X ├───────    lqr_1_1: ───┤ X ├───────────────
+                 ┌──┴───┴──┐           └───┘                    ┌──┴───┴──┐┌───┐
+        lqr_1_2: ┤ U1(0.1) ├  +                     =  lqr_1_2: ┤ U1(0.1) ├┤ X ├───────
+                 └─────────┘                                    └─────────┘└─┬─┘┌─────┐
+        lqr_2_0: ─────■─────                           lqr_2_0: ─────■───────■──┤ Tdg ├
+                    ┌─┴─┐                                          ┌─┴─┐        └─────┘
+        lqr_2_1: ───┤ X ├───                           lqr_2_1: ───┤ X ├───────────────
+                    └───┘                                          └───┘
+        lcr_0: 0 ═══════════                           lcr_0: 0 ═══════════════════════
+        lcr_1: 0 ═══════════                           lcr_1: 0 ═══════════════════════
+
+  With this, Qiskit's circuits no longer assume an implicit
+  initial state of :math:`|0\rangle`, and will not be drawn with this
+  initial state. The all-zero initial state is still assumed on a backend
+  when a circuit is executed.
```

Comment:
Since this goes to `rst` I used sections, but feel free of course to reformat as required
```suggestion
  when a circuit is executed.
  
    The Qiskit Aqua 0.7.0 release introduces a lot of new functionality along
    with an improved integration with ``QuantumCircuit`` objects. The central
    contributions are the optimization stack, a complete refactor on Operators,
    using circuits as native input for the algorithms and removal of the
    declarative JSON API.

    Optimization stack
    ==============
    The ``qiskit.optimization`` module now offers an application stack for
    solving quadratic programs. It provides various near-term quantum and
    conventional algorithms, such as the ``MinimumEigenOptimizer`` or
    ``CplexOptimizer``, and a set of converters to translate between different
    problem representations, such as ``QuadraticProgramToQubo``.
    See the
    [changelog](https://github.com/Qiskit/qiskit-aqua/blob/master/CHANGELOG.md)
    for a list of the added features.

    Operator flow
    ===========
    The operator logic provided in ``qiskit.aqua.operators`` was completely
    refactored and is now a full set of tools for constructing
    physically-intuitive quantum computations. It contains state functions,
    operators and measurements and internally relies on Terra's Operator
    objects. Computing expectation values and evolutions was heavily simplified
    and objects like the ``ExpectationFactory`` produce the suitable, most
    efficient expectation algorithm based on the Operator input type.
    See the
    [changelog](https://github.com/Qiskit/qiskit-aqua/blob/master/CHANGELOG.md)
    for a overview of the added functionality.


    Native circuits
    ===========
    Algorithms commonly use parameterized circuits as input, for example the
    VQE, VQC or QSVM. Previously, these inputs had to be of type
    ``VariationalForm`` or ``FeatureMap`` which were wrapping the circuit
    object. Now circuits are natively supported in these algorithms, which
    means any individually constructed ``QuantumCircuit`` can be passed to
    these algorithms. In combination with the release of the circuit library
    which offers a wide collection of circuit families, it is now easy to
    construct elaborate circuits as algorithm input.

    Declarative JSON API
    =================
    The ability of running algorithms using dictionaries as parameters as well
    as using the Aqua interfaces GUI has been removed.
  
```

Reply:
Thanks, using sections like this doesn't actually fit into the way reno works, it would have to be a subsection or a subsubsection (reno generates it's own sections, you can use `reno report` to generate the output rst). That being said these are for aqua and also don't fit into this prelude section here (which is only about terra). I'll save the text and integrate it when we update the release documentation as part of bumping the versions in the meta-package.